code/modules/antagonists/heretic/structures/knock_final.dm

```diff
@@ -0,0 +1,73 @@
+/obj/structure/knock_tear
+	name = "???"
+	desc = "It stares back. Theres no reason to remain. Run."
+	max_integrity = INFINITE
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	icon = 'icons/obj/anomaly.dmi'
+	icon_state = "bhole3"
+	color = "#53277E"
+	light_color = "#53277E" //cooler purple
+	light_range = 20
+	anchored = TRUE
+	density = FALSE
+	layer = HIGH_PIPE_LAYER //0.01 above sigil layer used by heretic runes
+	var/datum/mind/ascendee
+	///a static list of heretic summons, this shouldnt even matter enough to be static but whatever
+	var/static/list/monster_types
+
+/obj/structure/knock_tear/Initialize(mapload, ascendant)
+	. = ..()
+	transform *= 3
+	if(!monster_types)
+		monster_types = subtypesof(/mob/living/simple_animal/hostile/heretic_summon) - /mob/living/simple_animal/hostile/heretic_summon/armsy/prime
+	if(ascendant)
+		ascendee = ascendant
+		RegisterSignal(ascendee.current, COMSIG_ATOM_EXAMINE, PROC_REF(master_examine))
```

Comment:
This is fragile / breaks in the event of mind transfers, 
which normally I'd say is whatever 

but you give them a spell which makes them mind transfer regularly (shapechange)

Reply:
still works if the master changes to lets say a prophet and back, the examine text is still there